task4/task4.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+def movie_hash (file_name: "movies.txt")
+  review_file = CSV.read(file_name)
+  arr = []
+  CSV.foreach("movies.txt") do |row|
```

Comment:
Use CSV.read with "col_sep" and "headers" options instead. Check documentation https://docs.ruby-lang.org/en/master/CSV.html#class-CSV-label-Options+for+Parsing